test.js

```diff
@@ -168,6 +168,8 @@ test('`basePath` option', t => {
     at Object.<anonymous> (node_modules/foo/bar.js:1:14)
     at node_modules/foo/baz.js:1:14`;
 	t.is(cleanStack(stack, {basePath}), expected);
+	t.is(cleanStack(cleanStack(stack, {basePath})), expected);
+	t.is(cleanStack(cleanStack(stack, {basePath}), {basePath}), expected);
```

Comment:
Without this PR, those two assertions fail. This is what the PR is trying to fix.